src/styles/common/_colors.scss

```diff
@@ -209,3 +209,8 @@ $bookmarkPanelButtonBackgroundColor: #cca3f5;
 $bookmarkLinkHoverColor: #040505;
 $eventsListLoadButtonColor: #e5eaf0;
 $eventsListLoadButtonTextColor: #bc271a;
+
+$attachedMessageCardColor: #FFA64C;
```

Comment:
there is alread attachedMessageColor and other color variables from this one. Can we check if they are even used anywhere? if not let's delete then, otherwise its confusing that we have 2 attached message color variables